backend/plonk/bn254/solidity.go

```diff
@@ -1116,6 +1174,7 @@ contract PlonkVerifier {
         res := mload(mPtr)
       }
     }
+    success = true;
```

Comment:
Why this?

Reply:
an enormous mistake that made me think it was working while there was a bug^^ I fixed it last friday